caffe2/python/operator_test/top_k_test.py

```diff
@@ -97,7 +100,7 @@ def bind_ref(X_loc):
         self.assertReferenceChecks(gc, op, [X], bind_ref)
 
     @given(bs=st.integers(1, 3), n=st.integers(1, 10000), k=st.integers(1, 1),
-            flatten_indices=st.booleans(), **hu.gcs)
+           flatten_indices=st.booleans(), **hu.gcs)
     def test_top_k_2(self, bs, n, k, flatten_indices, gc, dc):
         X = np.random.rand(bs, n).astype(dtype=np.float32)
```

Comment:
Please set the seed using hypothesis package, it's useful for reproducing the problem.